tests/testsuite/fix.rs

```diff
@@ -1117,3 +1117,26 @@ fn doesnt_rebuild_dependencies() {
 ")
         .run();
 }
+
+#[test]
+fn does_not_crash_with_rustc_wrapper() {
+    // We don't have /usr/bin/env on Windows.
```

Comment:
So? It's expressly ignored, so whatever you set `RUSTC_WRAPPER` doesn't even need to exist on non-windows, no?